vivisect/tests/testvivisect.py

```diff
@@ -51,8 +51,8 @@ def checkTestWorkspace(self, vw, ans):
 
 
     def getAndTestWorkspace(self, fname):
-        vw = self.getTestWorkspace(fname)
-        ans = self.getAnsWorkspace(fname)
```

Comment:
these were moved out of this class into a more general test module over a year ago.